src/open_inwoner/configurations/tests/test_admin.py

```diff
@@ -19,89 +18,39 @@ class TestAdminSite(WebTest):
     def setUpTestData(cls):
         cls.user = UserFactory(is_superuser=True, is_staff=True)
 
-    def test_delete_site_success(self):
-        site2 = Site.objects.create(id=2, domain="example2.com", name="example2")
+    def test_site_add_prevented_if_one_exists(self):
+        """Test that adding a site is prevented if one already exists."""
+        # Make sure we have exactly one site
+        initial_site_count = Site.objects.count()
 
-        # delete site
         response = self.app.get(
-            reverse("admin:sites_site_delete", kwargs={"object_id": site2.id}),
-            user=self.user,
+            reverse("admin:sites_site_add"), user=self.user, expect_errors=True
         )
-        response = response.form.submit().follow()
 
-        # check sites
-        sites = Site.objects.all()
-        self.assertEqual(len(sites), 1)
+        self.assertEqual(response.status_code, 403)
 
-        # check message
-        doc = PyQuery(response.content)
+        self.assertEqual(Site.objects.count(), initial_site_count or 1)
 
-        success_msg = doc.find(".success").text()
-        self.assertIn("met succes verwijderd.", success_msg)
+    def test_site_deletion_prevented(self):
+        """Test that site deletion is prevented."""
+        # Make sure we have exactly one site
+        site = Site.objects.first()
```

Comment:
```suggestion
        site = Site.objects.get()
```

So it'll raise if it's not 1.